### PR TITLE
Add pod affinity config

### PR DESCRIPTION
**Pull request by shurshun**

### CODE REVIEW

File: Application.pkl [hunk 1]
```diff
@@ -78,6 +80,7 @@ deployment: Deployment = new {
         }
       }
       spec {
+        affinity = module.affinity.output.value
```
Comment (germanosin) [inline Application.pkl:83]: Why do we put yaml here?

Comment (germanosin) [reply Application.pkl:83]: I suppose this should be module.affinity.config ?

File: common/affinities.pkl [hunk 2]
```diff
@@ -0,0 +1,44 @@
+open module hpkl.common.k8s.affinities
+
+import "@k8s/api/core/v1/PodSpec.pkl"
+
+class PodAntiAffinityConfig {
+  preset: "soft"| "hard" = "soft"
```
[the remark this answers is not in the document]
Comment (shurshun) [reply common/affinities.pkl]: Switched on "Soft|Hard"

File: Application.pkl (hunk 1 above)
Comment (shurshun) [reply Application.pkl]: Yeap, fixed

**Review by germanosin — APPROVED** [review]

**Pull request merged**